more_itertools/recipes.py

```diff
@@ -1134,7 +1134,7 @@ def is_prime(n):
     This function uses the Miller-Rabin primality test, which can return false
     positives for very large inputs. For values of *n* below 10**24
     there are no false positives. For larger values, there is less than
-    a 1 in 2**64 false positive rate. Multiple tests can reduce the chance
+    a 1 in 2**128 false positive rate. Multiple tests can reduce the chance
```

Comment:
Perhaps while we're here, change "can" to "can further".   One in 2**128 is already so rare that it borders on "impossible".